content/_help/verify-your-identity/how-to-add-images-of-your-state-issued-id._es.md

```diff
@@ -1,32 +1,137 @@
+  type='error'
+  role='alert'
+%}
 
-* Sigue los mismos consejos para hacer fotos de tu documento de identidad con una cámara
+<div class="margin-top-5">
+  <span class="usa-tag usa-tag--informative">Recomendado</span>
+</div>
+
+<div class="margin-top-2">
+  <h2 id="phone-tips" class="margin-0">
+      Con un teléfono, realice fotografías del anverso y reverso de su documento de identidad
+  </h2>
+</div>
+
+Si está utilizando una computadora para verificar su identidad, puede cambiar a un teléfono para realizar fotografías de su documento de identidad.
+
+### Consejos para obtener fotografías nítidas con un teléfono
+
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    <img alt="El frente del documento de identidad está en el centro del marco de la cámara y sobre un fondo de tono oscuro." src="{{ site.baseurl }}/assets/img/help/id-dos-and-donts/id-do-front.png" />
+  </div>
+  <div class="tablet:grid-col">
+    <img alt="La parte posterior del documento de identidad está en el centro del marco de la cámara y sobre un fondo de tono oscuro." src="{{ site.baseurl }}/assets/img/help/id-dos-and-donts/id-do-back.png" />
+  </div>
+</div>
+
+{%
+  include components/icon-list.html
+  items=page.do_list
+  size='md'
+  icon_color='green'
+  icon_shape='check_circle'
+%}
+
+### Aspectos habituales al realizar fotografías de un documento de identidad
+
+<div class="grid-row grid-gap margin-bottom-6">
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='El documento de identidad se coloca sobre un fondo de color blanco.'
+      image='/assets/img/help/id-dos-and-donts/id-dont-white-background.png'
+      bordered_image=true
+      do_dont_text="NO"
+      text='Utilizar un fondo blanco'
+    %}
+  </div>
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='El documento de identidad se coloca sobre un fondo texturizado.'
+      image='/assets/img/help/id-dos-and-donts/id-dont-textured-background.png'
+      bordered_image=true
+      do_dont_text="NO"
+      text='Utilizar fondos con texturas o dibujos'
+    %}
+  </div>
+</div>
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='Los bordes del documento de identidad se encuentran recortados.'
+      image='/assets/img/help/id-dos-and-donts/id-dont-cropped.png'
+      bordered_image=true
+      do_dont_text="NO"
+      text='Recortar los bordes de la identificación'
+    %}
+  </div>
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='El documento de identidad presenta reflejos y sombras.'
+      image='/assets/img/help/id-dos-and-donts/id-dont-shadow.png'
+      do_dont_text="EVITE"
+      text='Brillos y sombras'
+    %}
+  </div>
+</div>
+
+<div class="page-content__horizontal-rule"></div>
```

Comment:
I might think it'd be fine to use an element selector within the `page-content__prose` context, so that you could then use markdown for this separator and avoid HTML:

```suggestion
---

```

Corresponding SCSS updates:

```scss
.page-content__prose hr {
  @include u-border-top(1px, 'base-lighter');
  @include u-margin-top(4);
}
```

I'm not aware that we're using HR's anywhere else to need to worry much about impact, though honestly if it did, I'd prefer ourselves to be forced to reconcile HR styling 😅

Reply:
Made that change. That's much neater.

I do see us using horizontal rules in the side nav (tablet size) and the footer, but as far as I can tell this change doesn't effect those uses.
<img width="744" alt="Screenshot 2023-09-11 at 10 34 53" src="https://github.com/18F/identity-site/assets/6818839/8cf63229-19a9-46ba-bdcb-3aa202eae79b">
<img width="617" alt="Screenshot 2023-09-11 at 10 34 49" src="https://github.com/18F/identity-site/assets/6818839/57f93c69-d0f8-40c2-9cf6-701ab77efd4d">